### PR TITLE
Updated Twitter logo 

### PR DESCRIPTION
**Pull request by adityadeshpande09**
fixes: #262 
image

The twitter logo was updated to X.

### CODE REVIEW

File: src/graphics/TwitterIcon/index.tsx [hunk 1]
```diff
@@ -2,17 +2,23 @@ import * as React from 'react'
...
+      viewBox="0 0 1200 1227"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <g>
+        <path
+          fill-rule="evenodd"
+          clip-rule="evenodd"
+          d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"
+          fill="white"
```
Comment (jacobsfletch) [inline src/graphics/TwitterIcon/index.tsx:17]:
```suggestion
          fill="var(--theme-bg)"
```

File: src/graphics/TwitterIcon/index.tsx [hunk 2]
```diff
@@ -2,17 +2,23 @@ import * as React from 'react'
...
+          stroke="white"
```
Comment (jacobsfletch) [inline src/graphics/TwitterIcon/index.tsx:18]:
```suggestion
          stroke="var(--theme-bg)"
```

**Comment by jacobsfletch** [comment]
Thank you @adityadeshpande09 👍 I left a couple change suggestions. Could you also remove the LICENSE from this PR and add it to another?

**Comment by jacobsfletch** [comment]
Oh I'm now seeing you do have another PR for the license already, good work, way ahead of me. Can you remove the changes to that file from this PR?

**Comment by adityadeshpande09** [comment]
> Oh I'm now seeing you do have another PR for the license already, good work, way ahead of me. Can you remove the changes to that file from this PR?

ohk works 👍

**Comment by adityadeshpande09** [comment]
@jacobsfletch  Is this fine ? I have removed the License file from this PR.

**Comment by jacobsfletch** [comment]
@adityadeshpande09 NICE, appreciate your speed on this. Looks like there's just one last change to be made, which is to make the svg stroke match the fill.

**Comment by adityadeshpande09** [comment]
Let me know if there are any changes 😄

**Comment by jacobsfletch** [comment]
Thanks @adityadeshpande09 👏

**Pull request merged**